packages/components/src/components/hds/filter-bar/filter-group/radio.hbs

```diff
@@ -0,0 +1,22 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<li class="hds-filter-bar__filter-group__radio hds-filter-bar__filter-group__selection-option">
```

Comment:
Suggestion: instead of using uls/lis to group the inputs together - I think it would be better to use fieldset + legend.

the legend could be visually hidden and just say "filter by {filter name}"